README.md

```diff
@@ -90,6 +90,18 @@ multiple({
 ```
 
 ### Effective Odds
+#### One selection
+
+```
```

Comment:
You are able to add a label of javascript for nice highlighting in Markdown. 
This line will then become:
"```javascript"